documentation/tutorial.md

```diff
@@ -68,3 +68,145 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
+
+For this part of the Tutorial we will access MongoDB - via the MongoDB shell `mongosh`. You can read more about the MongoDB shell [here](https://www.mongodb.com/docs/mongodb-shell/).
+
+
+### Install the MongoDB shell:
+While MongoDB is installed within the LXD containers that the application is hosted on, it is not installed on your machine. This means that you will have to install the MongoDB shell yourself. Install `mongosh` by following the official [instructions on how to install the MongoDB shell](https://www.mongodb.com/docs/mongodb-shell/install/#std-label-mdb-shell-install).
+
+### MongoDB URI:
+Connecting to the database requires a [URI](https://www.mongodb.com/docs/manual/reference/connection-string/). We use a URI of the format:
+```
+mongodb://<username>:<password>@<hosts>/<database name>?replicaSet=<replica set name>
+```
+
+Connecting via the URI requires that you know the values for `username`, `password`, `hosts`, `database name`, and the `replica set name`. We will show you how to retrieve the necessary fields. 
+
+#### Retrieving the username:
+In this case, we are using the `admin` user to connect to MongoDB. Use `admin` as the username.
+
+#### Retrieving the password:
+The password can be retrieved by running the `get-admin-password` action on the Charmed MongoDB application:
+```
+juju run-action mongodb/leader get-admin-password --wait
+```
+Running the command should output:
+```
+unit-mongodb-0:
+  UnitId: mongodb/0
+  id: "2"
+  results:
+    admin-password: <password>
+  status: completed
+  timing:
+    completed: 2022-12-02 11:30:01 +0000 UTC
+    enqueued: 2022-12-02 11:29:57 +0000 UTC
+    started: 2022-12-02 11:30:01 +0000 UTC
+```
+Use the password under the result: `admin-password`.
+
+#### Retrieving the hosts:
+The hosts are the units hosting the MongoDB application. In this case we only have one host; you can find the host’s IP address with `juju status`. This will output information about the Charmed MongoDB application along with it’s IP address:
+```
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  11:31:16Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      1  mongodb  dpe/edge   96  no       Replica set primary
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        <host IP>    27017/tcp  Replica set primary
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+
+```
+Use the IP address listed underneath `Public address` for `mongodb/0` as your host.
+
+#### Retrieving the database name:
+In this case we are logging in via the `admin` user so we will be connecting to the `admin` database. Use `admin` as the database name.
```

Comment:
```suggestion
In this case we are logging in via the `admin` user so we will be connecting to the `admin` database. Use `admin` as the database name. We will in the following create a "test" database where to store some data. 
```

Reply:
I don't think this is the correct place for this. What do you think about putting this further down on line 167

Reply:
yes. But, I mean, but when I "tell a story", sometimes I believe it is good to create anticipations and connect different part of the story. This works on many things I have been working on: creating a pitch, a presentation or (maybe) a tutorial. It should not however distract the reader/audience but rather get him interested. I believe this was a good point to do that, but it is not a strong point, if you and @WRFitch feel otherwise. Beside, I really don't know how Daniele or Nick would feel about this, did not double check this with them to be fair.

Reply:
Can we get a second opinion on this @WRFitch and @zmraul 🙏

Reply:
I don't have an opinion on this. I think it's just personal preference.